src/test/rustdoc-js-std/filter-crate.js

```diff
@@ -1,6 +1,6 @@
 // exact-check
 
-const QUERY = 'hashmap';
+const QUERY = '"hashmap"';
```

Comment:
Why was this diff needed?

Reply:
This is why:

![Screenshot from 2022-01-04 15-52-35](https://user-images.githubusercontent.com/3050060/148077442-ec1906ef-595c-4914-b003-6710cd3313d5.png)

Reply:
The screenshot doesn't clarify things for me. :-)

Reply:
Well, before they were rejecting but I don't why since the lev distance should be 3, which is less than 6 so they all should have been "accepted".

Reply:
What was being rejected?

Reply:
All the current results. Before, this query was returning nothing.

Reply:
I'm afraid I still don't understand. Here are the current results on doc.rust-lang.org:

https://doc.rust-lang.org/nightly/std/index.html?search=hashmap
```
std::collections::HashMap
std::collections::hash_map::HashMap
std::collections::hash_map
```

https://doc.rust-lang.org/nightly/std/index.html?search=%22hashmap%22
```
std::collections::HashMap
std::collections::hash_map::HashMap
```

Those seem correct. Here are the results from your demo:

https://rustdoc.crud.net/imperio/improve-rustdoc-search/std/index.html?search=hashmap
```
alloc::borrow::Cow::hash
alloc::boxed::Box::hash
alloc::collections::btree_map::BTreeMap::hash
alloc::collections::btree_set::BTreeSet::hash
alloc::collections::linked_list::LinkedList::hash
alloc::collections::vec_deque::VecDeque::hash
alloc::fmt::Error::hash
alloc::rc::Rc::hash
alloc::string::String::hash
alloc::sync::Arc::hash
alloc::vec::Vec::hash
```

https://rustdoc.crud.net/imperio/improve-rustdoc-search/std/index.html?search=%22hashmap%22
```
(nothing)
```

Those seem wrong.

Reply:
Did you set the crate filter on `core` (it's set in `FILTER_CRATE` just under this line)?